ci/install.sh

```diff
@@ -11,18 +11,11 @@ if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
   export PATH="/usr/local/opt/ccache/libexec:$PATH"
 fi
 
+# Use default CC to avoid compilation problems when installing Python modules.
 echo "Install neovim module for Python 3."
-# Allow failure. pyenv pip3 on travis is broken:
-# https://github.com/travis-ci/travis-ci/issues/8363
-CC=cc python3 -m pip -q install --user --upgrade neovim || true
-
-if ! [ "${TRAVIS_OS_NAME}" = osx ] ; then
-  # Update PATH for pip.
-  export PATH="$(python2.7 -c 'import site; print(site.getuserbase())')/bin:$PATH"
-  # Use default CC to avoid compilation problems when installing Python modules.
-  echo "Install neovim module for Python 2."
-  CC=cc python2.7 -m pip -q install --user --upgrade neovim
-fi
+CC=cc pip3 -q install --user --upgrade neovim
+echo "Install neovim module for Python 2."
+CC=cc pip2 -q install --user --upgrade neovim
```

Comment:
I do not understand this change. The `-m pip` form was very intentional...

Reply:
It is just more straight forward IMHO - it really ensures that pip is also installed in the path.

Reply:
`python -m pip` is the python-recommended way of invoking `pip[3]`. Using `pip[3]` directly is deprecated, and  imprecise because different versions of python may exist on a system.

When reverting changes that were made previously (see `git log`) it is good to at least discuss it explicitly with the previous author.

Reply:
Ok, will change that.